deploy_heroku.py

```diff
@@ -212,7 +212,12 @@ def get_git_push_spec():
 while not redis_provisioned:
     time.sleep(30)
     redis_output = subprocess.check_output(redis_status_command).decode().split("\n")
-    redis_provisioned = "available" in redis_output[2]
+    for status in redis_output:
+        match = re.match(r"^Status:\s+available",status)
```

Comment:
```suggestion
        match = re.match(r"^Status:\s+available", status)
```
Linting is complaining